src/vs/workbench/services/runtimeStartup/common/runtimeStartup.ts

```diff
@@ -182,6 +186,16 @@ export class RuntimeStartupService extends Disposable implements IRuntimeStartup
 			}
 		}));
 
+		// Add the onDidCompleteDiscoveryPhase event handler.
+		this._register(this.onDidCompleteDiscoveryPhase(() => {
+			if (this._languageRuntimeService.registeredRuntimes.length === 0) {
+				// if no runtimes were found, notify the user about the problem
+				this._notificationService.error(nls.localize('noRuntimesMessage',
```

Comment:
Do we have any standards for how we like to name our localization keys? It doesn't seem like we're actually concerned about localization for now and this may all be revisited later but if there's a method to naming this that we like to use, please let me know!

Reply:
You're right. Localization keys are not standardized in the app. We need to work on this. In general, I have tried to name many of my keys with a prefix of `positron.` so that they are namespaced and don't collide with VSCode.

```
// Get the title and message.
const title = localize('positron.dataExplorer.selectColumn', "Select Column");
const message = localize(
	'positron.dataExplorer.unableToOpenTheColumnSelector',
	"Unable to open the column selector."
);
```

But I have not done this perfectly.